src/components/encryption.ts

```diff
@@ -86,16 +86,18 @@ export class EncryptedEventBroker {
         private onEphemeralEvent: ((event: EphemeralEvent) => void)|undefined,
         private getIntent: (userId: string) => Intent
         ) {
+        if (this.onEphemeralEvent) {
+            this.presenceCleanupInterval = setInterval(() => {
+                const ts = Date.now() - 30000;
```

Comment:
30s wants to be a config var?

Reply:
We should make it a const but I don't think there is any value in it, it's just a cache lifetime.